scripts/fetch-entity/src/drupalClient.ts

```diff
@@ -5,34 +5,13 @@
  */
 
 import { DrupalClient } from 'next-drupal'
-
-/* eslint-disable import/no-extraneous-dependencies */
-import { SocksProxyAgent } from 'socks-proxy-agent'
-import fetch from 'node-fetch'
-
-const syswideCas = await import('syswide-cas')
-syswideCas.addCAs('certs/VA-Internal-S2-RCA-combined.pem')
-syswideCas.addCAs('certs/rootCA.pem')
+import { getFetcher } from 'proxy-fetcher'
```

Comment:
Simplified this to use `proxy-fetcher`

Reply:
It's a good question. I didn't want to change too much at once, but I can try it now